modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -301,21 +305,31 @@ object ElasticRequest {
       self.copy(routing = Some(value))
   }
 
-  sealed trait SearchRequest extends ElasticRequest[SearchResult]
+  sealed trait SearchRequest extends ElasticRequest[SearchResult] with WithSort[SearchRequest]
 
   private[elasticsearch] final case class Search(
     index: IndexName,
     query: ElasticQuery[_],
+    sortBy: Option[List[SortBy]],
     routing: Option[Routing]
-  ) extends SearchRequest
+  ) extends SearchRequest { self =>
+    def sortBy(sorts: SortBy*): SearchRequest =
+      self.copy(sortBy = Some(sorts.toList))
```

Comment:
Does it make sense to append to existing sorts? Also, does it make sense to use `Set`?